.aoc-stack-test.yml

```diff
@@ -4,4 +4,4 @@ s3ReleaseCandidateBucketName: "aws-otel-collector-release-candidate"
 s3BucketName: "aws-otel-collector-test"
 sshKeyS3BucketName: "aoc-ssh-key"
 traceDataS3BucketName: "trace-expected-data"  
-testingImageRepoName: "611364707713.dkr.ecr.us-west-2.amazonaws.com/aws/aws-otel-collector"
+testingImageRepoName: "public.ecr.aws/aws-otel-test/adot-collector-integration-test"
```

Comment:
What is this used for?

Reply:
I assume its purpose is for notifying what [resources are used with the release and testing](https://github.com/aws-observability/aws-otel-collector/blob/main/.github/workflows/CI.yml#L476-L479). Currently ask Ying to confirm about it.

Reply:
Got a confirmation from @wyTrivail. So it's purpose is notify what resources are being used for the validation test and release validation. An example would be https://github.com/aws-observability/aws-otel-collector/blob/main/.github/workflows/CD.yml#L187. Somehow we decide to define it explicitly instead of using a yaml file anymore. Seems weird for me to delete it since most of the release validation or integration test, its used the same bucket so either define a yaml or define an environment variable is better for me instead of defining it explicitly though.